### ISSUE TITLE
[4.0.0-beta.16]: ion-checkbox breaks completly in IE11

### ISSUE THREAD

**Issue by majorbenks** [ionic-team/ionic#16439]
# Bug Report

**Ionic Info**

```
Ionic:

   ionic (Ionic CLI)             : 4.3.1 (C:\Users\dst\AppData\Roaming\npm\node_modules\ionic)
   Ionic Framework               : @ionic/angular 4.0.0-beta.15
   @angular-devkit/build-angular : 0.10.3
   @angular-devkit/schematics    : 7.0.3
   @angular/cli                  : 7.0.3
   @ionic/angular-toolkit        : 1.0.0

Cordova:

   cordova (Cordova CLI) : not installed
   Cordova Platforms     : not available
   Cordova Plugins       : not available

System:

   NodeJS : v10.13.0 (C:\Program Files\nodejs\node.exe)
   npm    : 6.4.1
   OS     : Windows 10
```

**Describe the Bug**
Since 4.0.0-beta.16 the ion-checkbox is no longer displayed in IE11. There isn't even any HTML that could be styled, the ion-checkbox tag has only an hidden input as child-element.

**Steps to Reproduce**
put a ion-checkbox and a ion-label inside a ion-item.

**Related Code**
4.0.0-beta.15 gives me this HTML in IE11 dev-tools:
`<ion-checkbox class="ng-untouched ng-pristine ng-valid sc-ion-checkbox-md-h ion-color ion-color-secondary in-item interactive hydrated" slot="start" color="secondary"><div class="checkbox-icon sc-ion-checkbox-md"><div class="checkbox-inner sc-ion-checkbox-md"></div></div><input name="ion-cb-21" class="sc-ion-checkbox-md" id="ion-cb-21" aria-labelledby="ion-cb-21-lbl" type="checkbox" value="on"></ion-checkbox>`
This is more or less the same as Chrome dev-tools shows, just without shadow dom (as IE11 doesn't support shadow dom). However, I can style that and make it work in IE.

4.0.0-beta.16 gives me this HTML in IE11 dev-tools:
`ion-checkbox class="ng-untouched ng-pristine ng-valid ion-untouched ion-pristine ion-valid sc-ion-checkbox-md-h ion-color ion-color-secondary in-item checkbox-checked interactive hydrated" role="checkbox" aria-checked="true" aria-labelledby="ion-cb-139-lbl" slot="start" color="secondary" ng-reflect-color="secondary" ng-reflect-model="true" ng-reflect-disabled="false" ng-reflect-is-disabled="false" ng-reflect-checked="true"><input name="ion-cb-139" class="aux-input" type="hidden" value="true"></ion-checkbox>`
Here everything that's inside shadow dom (as shown in Chrome dev-tools) is missing in IE11. Therefore I don't have elements to style. (Chrome shows a svg and a button inside the shadow dom. That is missing in IE11 dev-tools.)

**Expected Behavior**
I think the elements inside the shadow dom should somehow be made known to IE11 as it was in 4.0.0-beta.15, so that can be styled and used in IE11 too. (it's still quite some styling work, but without elements, I can't style anything.)

**Comment by manucorporat** [comment]
Next release should fix most of the issue with Android 4.4 and IE11! Thanks for the issue!

**Comment by majorbenks** [comment]
I'm sorry to say, but this issue is still in beta.17 - and it actually got worse: ion-radio has now the same issue: shadow-dom html is not rendered and therefore missing. This means there is no html element that could be styled as anything close to a checkbox / radio.
